CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -1679,18 +1679,34 @@ CFTimeInterval CFTimeZoneGetDaylightSavingTimeOffset(CFTimeZoneRef tz, CFAbsolut
     return 0.0;
 }
 
+extern UCalendar *__CFCalendarCreateUCalendar(CFStringRef calendarID, CFStringRef localeID, CFTimeZoneRef tz);
```

Comment:
should this be CF_PRIVATE? what's the tag on that function?

Reply:
I'm not sure, it was just moved up from below in the same file.  Pretty sure its meant to be private.

Reply:
My bad, I just saw the removal under it.